src/swell/utilities/run_jedi_executables.py

```diff
@@ -112,14 +112,19 @@ def run_executable(
     np: int,
     jedi_executable_path: str,
     jedi_config_file: str,
-    output_log: str
+    output_log: str,
+    perhost: int=None
```

Comment:
To satisfy the type checker (otherwise, it complains that `None` cannot be coerced to type `int`).

```suggestion
    perhost: Optional[int]=None
```